examples/echobot.py

```diff
@@ -34,7 +34,7 @@ def start(update: Update, _: CallbackContext) -> None:
     """Send a message when the command /start is issued."""
     user = update.effective_user
     update.message.reply_markdown_v2(
-        f'Hi {user.mention_markdown_v2()}\!',
+        f'Hi {user.mention_markdown_v2()}!',
```

Comment:
the `!` needs to be escaped for MarkdownV2. Please instead make it a raw string. Apparently I missed that in the latest commit to master …